experimenter/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/index.tsx

```diff
@@ -298,7 +298,18 @@ export const FormBranches = ({
               type="checkbox"
               isInvalid={!!fieldMessages?.is_rollout}
               feedback={fieldMessages?.is_rollout}
-              label="This is a rollout (single branch)"
+              label={
+                <>
+                  This is a rollout (single branch).{" "}
+                  <a
+                    href="https://experimenter.info/deep-dives/experimenter/rollouts"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    Learn more
```

Comment:
Awesome thank you @KramStyles 🎉

Reply:
The pleasure is all mine @jaredlockhart. I'm on standby still waiting to be assigned to more tasks.